common/credential-storage/src/ephemeral_storage.rs

```diff
@@ -53,31 +53,41 @@ impl Storage for EphemeralStorage {
 
     async fn get_next_unspent_credential(
         &self,
-        gateway_id: &str,
     ) -> Result<Option<StoredIssuedCredential>, Self::StorageError> {
         // first try to get a free pass if available, otherwise fallback to bandwidth voucher
         let maybe_freepass = self
             .coconut_credential_manager
-            .get_next_unspect_freepass(gateway_id)
+            .get_next_unspent_freepass()
             .await;
         if maybe_freepass.is_some() {
             return Ok(maybe_freepass);
         }
 
         Ok(self
             .coconut_credential_manager
-            .get_next_unspect_bandwidth_voucher()
+            .get_next_unspent_ticketbook()
             .await)
     }
 
-    async fn consume_coconut_credential(
+    async fn consume_coconut_credential(&self, id: i64) -> Result<(), StorageError> {
```

Comment:
ecash

Reply:
It's actually unused

Reply:
Removed in 098a45a7b304f66e4c224ec8843ba2b1ea172332